tox.ini

```diff
@@ -5,7 +5,6 @@ skip_missing_interpreters = true
 [testenv:pylint]
 deps =
    git+https://github.com/pycqa/astroid@master
-   isort
```

Comment:
I was starting a comment sounding like: "But what if pytest remove isort from its dependencies?". But after reading the commit in detail, I think there is a typo in your commit message "isort is already a dependency of pytest" should probably be "isort is already a dependency of pylint" instead.

Reply:
That's indeed a typo.